frontend/src/main-components/ImageInterfaceContent.css

```diff
@@ -0,0 +1,82 @@
+.Container {
+  flex: 9;
+  display: flex;
+  border: 1rem solid white;
+  border-radius: 1rem;
+  background-color: white;
+  margin: 1%;
+  flex-direction: column;
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.5rem;
+}
```

Comment:
Google style guide says to put a space between rules. https://google.github.io/styleguide/htmlcssguide.html#CSS_Formatting_Rules